src/Cookie/SessionCookieJar.php

```diff
@@ -70,7 +70,7 @@ protected function load(): void
             foreach ($data as $cookie) {
                 $this->setCookie(new SetCookie($cookie));
             }
-        } elseif (\strlen($data)) {
+        } elseif (!\is_string($data) || \strlen($data)) {
```

Comment:
This could potentially be wrong because `$data` may still be `bool|int|float|string` at this point, though it seems like the intended effect as introduced in 2ba19b2c8b22794071f6053db2c43773aff06e59 means it could be simplified to:

```suggestion
        } elseif ($data) {
```

(or simply an `else`)